plugins/modules/ndo_interface_setting.py

```diff
@@ -578,7 +580,7 @@ def main():
                 ops.append(dict(op="remove", path=interface_path + "/domains"))
                 proposed_payload.pop("domains", None)
             elif domains:
-                domain_uuid_list = validate_domains(mso, domains, template, template_info)
+                domain_uuid_list = validate_domains(mso, domains, template, template_info)[0]
```

Comment:
Isn't it clearer to do something like:
```suggestion
                (domain_uuid_list, ) = validate_domains(mso, domains, template, template_info)
```

Reply:
I get `ValueError: too many values to unpack (expected 1)` when I try to unpack values using `(domain_uuid_list, )`. 
Additionally, the syntax `(domain_uuid_list, *_ )` only works with Python 3 and not with Python 2.

Reply:
Done